tests/FSharp.Data.GraphQL.IntegrationTests/ResponseJsonTests.fs

```diff
@@ -0,0 +1,73 @@
+module FSharp.Data.GraphQL.IntegrationTests.ResponseJsonTests
+
+open System
+open Xunit
+open Helpers
+open FSharp.Data.GraphQL
+open FSharp.Data.GraphQL.Client
+
+// Local provider should be able to be created from local introspection json file.
+// This schema is adjusted to have some less common type names in order to test them.
+type Provider = GraphQLProvider<"introspection-altered-types.json">
```

Comment:
@hth313 I moved it to integration tests as we do not reference DesignTime dll in tests not to lock it by VS